pages/opportunities/[slug].tsx

```diff
@@ -83,16 +80,16 @@ const Page: NextPage<PageProps> = (props) => {
                     </a>
                   )}
               </S.OpportunityMetaRow>
-              <OpportunityMetaRow>
+              <S.OpportunityMetaRow>
                 <TimeIcon />
                 <Body>{opportunity.timeRequirements}</Body>
-              </OpportunityMetaRow>
-              <S.OpportunityOwnerWrapper>
+              </S.OpportunityMetaRow>
+              <S.OpportunityOwnerWrapper className="owner">
```

Comment:
To `className` je jen nějaký testovací pohrobek?

Reply:
Na tu třídu se právě odkazuju v tom testu: https://github.com/cesko-digital/web/blob/c48a3340df750ea7fc79b0cff15f21d251f1ef77/tests/e2e/opportunity.spec.ts#L5

Reply:
🤦 Jasně! Díky.

Reply:
(Pokud budem dávat třídy jen kvůli testům, zvážil bych nějaký prefix pro tyhle účely, ať nás to vždycky trkne.)